tsfc/driver.py

```diff
@@ -266,11 +273,14 @@ def compile_expression_at_points(expression, points, coordinates, parameters=Non
     if extract_arguments(expression):
         return ValueError("Cannot interpolate UFL expression with Arguments!")
 
+    # determine if we're in complex mode
+    complx = parameters["scalar_type"] == 'double complex'
```

Comment:
This is different logic to the one implemented in `compile_form`, which just checks for `"complex"`.